### ISSUE TITLE
Capcha using names for blocks that don't make sense

### ISSUE THREAD

**Issue by MyHomeAddress** [EarthMC/Issue-Tracker#1202]
When I joined the server today the captcha wanted me to identify the block mentioned, except it didn't make sense as it wanted me to identify "Stone slab2". There's no such thing as a Stone slab2 so I guessed red sandstone slab and apparently I was right. It was really confusing.
### Describe the solution you'd like
Could you change the names of how to identify blocks? I don't know if other blocks are affected, but they should change it as soon as possible because people might have a hard time joining the server when they don't know what they're doing.

Here's what it looked like for me:
![2020-07-25_11 18 44](https://user-images.githubusercontent.com/66042623/88460747-73a1e100-ce6c-11ea-8e62-f1f742c518f0.png)

**Comment by unbatedaura409** [comment]
It probably has something to do with the way minecraft names things. For example, when I used to play with worldedit a lot on freebuild servers years ago, I would need to use a list of every block ID. Back then, the block IDs were simple like that. Regular stone would be assigned the tag "minecraft:stone" and then be assigned a number, for regular stone it was 1. For granite, it would have a different tag, but it would use the number 1:1. Do you see what I'm getting at? Maybe the plugin is a tad outdated

**Comment by 32Vache** [comment]
> It probably has something to do with the way minecraft names things. For example, when I used to play with worldedit a lot on freebuild servers years ago, I would need to use a list of every block ID. Back then, the block IDs were simple like that. Regular stone would be assigned the tag "minecraft:stone" and then be assigned a number, for regular stone it was 1. For granite, it would have the same tag (minecraft:stone), but it would be assigned the number 1:1. Do you see what I'm getting at? Maybe the plugin is a tad outdated

This is true. 1.13 removed block numbered ids in favor of namespaced ids. the extra block data (like you said granite = 1:1) was also removed (eg changing stone:1 to minecraft:granite, things like leaves became minecraft:<variety>_leaves, ect)
In the case of slabs it also used a :id, but now it is <variety>_slab.

TLDR The plugin uses old blocks id that are no longer supported on 1.13+, making this glitch on the captcha

**Comment by Warriorrrr** [comment]
Closing this, captcha got removed ages ago so I guess this is 'solved'.
